extensions/do-not-disturb/package.json

```diff
@@ -42,6 +43,17 @@
       "mode": "no-view"
     }
   ],
+  "preferences": [
+    {
+      "name": "supressHUD",
+      "title": "Suppress HUD",
+      "label": "Suppress HUD notification when toggling Do Not Disturb",
+      "description": "This option exists for compatibility with other extensions.",
+      "type": "checkbox",
+      "defaultValue": false,
+      "required": false
+    }
+  ],
```

Comment:
The preference option is unnecessary. Simply load the `suppressHUD` from `launchContext`.